src/Workspaces/Remote/ServiceHub/Host/SolutionCreator.cs

```diff
@@ -24,14 +25,18 @@ namespace Microsoft.CodeAnalysis.Remote
     /// </summary>
     internal sealed class SolutionCreator
     {
+#pragma warning disable IDE0052 // used only in DEBUG builds
+        private readonly HostServices _hostServices;
+#pragma warning restore
```

Comment:
maybe put in `#if DEBUG`?

Reply:
Can't without introducing more #if DEBUG or warnings. The constructor takes it as an argument and assigns it.